### ISSUE TITLE
Expose ALB Listener's SSL Policy As A Manifest Parameter

### ISSUE THREAD

**Issue by adrianosela** [aws/copilot-cli#5310]
## Expose ALB Listener's SSL Policy As A Manifest Parameter

I want to restrict the minimum TLS version that my clients talk to my service with... for compliance-related reasons.
The problem is that copilot does not expose the ALB Listeners' Security Policy as manifest parameters I can set. So I am forced to resort to using YAML Patches.

Here's my YAML Patch:

```
# Enforce a modern TLS policy for the Application Load Balancer
- op: add
  path: /Resources/HTTPSListener/Properties/SslPolicy
  value: ELBSecurityPolicy-TLS13-1-2-2021-06
```

Copied from @tjhorner's comment in https://github.com/aws/copilot-cli/issues/3280.

**Comment by huanjani** [comment]
Thanks so much for opening this as a separate issue, @adrianosela!

I've marked it as a customer feature request, but to be transparent, please don't hold your breath for this as a manifest field--we are focused on unblocking users without workarounds like YAML patch/CDK overrides. Thanks for understanding!

**Comment by tim-xergy** [comment]
@huanjani I'd like to bump this up, having to manually make changes to the load balancer on any new environment is frustrating especially for security concerns like this

Could we at least have the default policy updated to one that disabled 1.0 and 1.1?

**Comment by proof-nicholas** [comment]
Agreed, I don't see any reason why the default policy should be 1.0.  We got dinged in a security scan because I assumed the tool would configure reasonable security defaults.
